packages/block-library/src/latest-posts/edit.js

```diff
@@ -72,47 +70,97 @@ class LatestPostsEdit extends Component {
 		this.isStillMounted = false;
 	}
 
-	toggleDisplayPostDate() {
-		const { displayPostDate } = this.props.attributes;
+	updateAttribute( name, value ) {
 		const { setAttributes } = this.props;
 
-		setAttributes( { displayPostDate: ! displayPostDate } );
+		setAttributes( { [ name ]: value } );
 	}
 
+// Difference between rangecontrol and querycontrol onNumberOfItemsChange??
+
 	render() {
 		const { attributes, setAttributes, latestPosts } = this.props;
 		const { categoriesList } = this.state;
-		const { displayPostDate, align, postLayout, columns, order, orderBy, categories, postsToShow } = attributes;
+		const { displayFullPost, displayPostContent, displayPostFeaturedImage, displayPostDate, displayPostAuthor, align, postLayout, columns, order, orderBy, categories, postsToShow, wordsToShow } = attributes;
 
 		const inspectorControls = (
 			<InspectorControls>
-				<PanelBody title={ __( 'Latest Posts Settings' ) }>
+				<PanelBody title={ __( 'Display Settings' ) }>
 					<QueryControls
-						{ ...{ order, orderBy } }
-						numberOfItems={ postsToShow }
-						categoriesList={ categoriesList }
-						selectedCategoryId={ categories }
-						onOrderChange={ ( value ) => setAttributes( { order: value } ) }
-						onOrderByChange={ ( value ) => setAttributes( { orderBy: value } ) }
-						onCategoryChange={ ( value ) => setAttributes( { categories: '' !== value ? value : undefined } ) }
 						onNumberOfItemsChange={ ( value ) => setAttributes( { postsToShow: value } ) }
 					/>
-					<ToggleControl
-						label={ __( 'Display post date' ) }
-						checked={ displayPostDate }
-						onChange={ this.toggleDisplayPostDate }
-					/>
 					{ postLayout === 'grid' &&
 						<RangeControl
 							label={ __( 'Columns' ) }
 							value={ columns }
 							onChange={ ( value ) => setAttributes( { columns: value } ) }
 							min={ 2 }
 							max={ ! hasPosts ? MAX_POSTS_COLUMNS : Math.min( MAX_POSTS_COLUMNS, latestPosts.length ) }
-							required
 						/>
 					}
 				</PanelBody>
+
+				<PanelBody title={ __( 'Featured Image Settings' ) }>
+					<ToggleControl
+						label={ __( 'Featured image' ) }
+						checked={ displayPostFeaturedImage }
+						onChange={ ( value ) => this.updateAttribute( 'displayPostFeaturedImage', value ) }
+					/>
+				</PanelBody>
+
+				<PanelBody title={ __( 'Post Content Settings' ) }>
+					<ToggleControl
+						label={ __( 'Post Content' ) }
+						checked={ displayPostContent }
+						onChange={ ( value ) => this.updateAttribute( 'displayPostContent', value ) }
+					/>
+					{ displayPostContent &&
+					<RadioControl
+        		label="Show:"
```

Comment:
is there an indentation problem here or is Github's problem?

Reply:
I m pretty sure that's a Github problem. It looks fine in my editor and didn't raise any js lint errors when I was testing